.github/workflows/BibtoolCI.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           sudo apt-get install -y bibtool
       - name: Execute bibtool and check for changes
         run: |
-          bibtool docs/oscar_references.bib -o docs/oscar_references.bib
+          bibtool docs/oscar_references.bib -o docs/oscar_references.bib |& bibtool.log
```

Comment:
This is supposed to write stdout and stderr both to the console and to a file. Console so it is visible in the log. File to make the `git status` below fail in case that bibtool printed something.
I guess this syntax is only supported by my zsh locally and not the shell on github actions. I'll try to change this

Reply:
Tests will already fail if there is any change to the `.bib` file. What other log message that neither correspond to a change in the `.bib` file nor to a non-zero exit code is this meant to catch?

Reply:
Unfortunately, the `check.error.rule` only print an error message to stderr, but the exit code is still 0. If you have a better idea than writing stderr to a file and check that file for being empty, please let me know.

Reply:
This should now be fixed. It first commits the empty `bibtool.log` file, then runs bibtool with pipes and afterwards checks `bibtool.log` for changes